edgelet/edgelet-docker/src/error.rs

```diff
@@ -73,6 +73,9 @@ pub enum ErrorKind {
     #[fail(display = "Invalid socket URI: {:?}", _0)]
     InvalidSocketUri(String),
 
+    #[fail(display = "Invalid socket URI")]
```

Comment:
Is this message correct? Seems to be copied from above.

Reply:
Good catch, fixed.